### PR TITLE
Add Miss Chance to Improvised Weapons for Throwing

### PR DESCRIPTION
**Pull request by Turtlelord26**
per #810

### CODE REVIEW

File: Items/04_01_Melee_&_Thrown_Weapons.txt [hunk 1]
```diff
@@ -21,26 +21,45 @@ Damage: 30 + STR mod + DEX mod Ballistic, minimum 10
 Range: Adjacent (1m)
 Accuracy Modifier: 0
 
+	Ranged Miss Chance
+
+* 2 out to 5m
+* 5 out to 20m
```
Comment (bleehu) [inline Items/04_01_Melee_&_Thrown_Weapons.txt:27]: did we want to add strength scaling for how far a thrown improvised weapon will go?

File: Items/04_01_Melee_&_Thrown_Weapons.txt [hunk 2]
```diff
@@ -21,26 +21,45 @@ Damage: 30 + STR mod + DEX mod Ballistic, minimum 10
...
 
+	Ranged Miss Chance
+
+* 2 out to 5m
+* 5 out to 20m
+
 	Some improvised weapons may have alternate damage types, at the DM's 
-discretion.
+discretion. Note that the miss chance stat is only relevant when throwing an 
+improvised weapon, while the accuracy is only used for melee attacks.
```
Comment (bleehu) [inline Items/04_01_Melee_&_Thrown_Weapons.txt:31]: great clarification! I love how I can understand these rules well the first time I read them. That wasn't always true with CX rules.

**Review by bleehu — APPROVED** [review]
Some minor suggested tweaks, but this would play very well!

File: Items/04_01_Melee_&_Thrown_Weapons.txt (hunk 1 above)
Comment (Turtlelord26) [reply Items/04_01_Melee_&_Thrown_Weapons.txt:27]: We could, but I have a suspicion that most of the throwing weapon fantasy is associated with dexy-assassin-type characters, and so making throwing weapons' full potential rely on strength would feel bad. We already use strength scaling on the damage, and that satisfies me on the flavor front.

**Pull request merged**